Python/Delta/GetAdvertisersDeltaGQL.py

```diff
@@ -0,0 +1,200 @@
+#################################################################
+# This script will retrieve all advertisers delta for a partner.
```

Comment:
This script will retrieve all advertiser deltas for a partner.

Reply:
Thanks! Fixed!